src/announcements/converters.ts

```diff
@@ -44,6 +44,7 @@ export const mapToAnnouncement = (
   if (!isAppPlatformValid(platforms)) return;
   if (appVersionMin && appVersionMin > APP_VERSION) return;
   if (appVersionMax && appVersionMax < APP_VERSION) return;
+  if (!actionButton) return;
```

Comment:
Are we sure that we never want to show announcements without an actionButton?

Reply:
I have discussed it with @gorandalum and to prevent user error or fallbacks, we should just remove them entirely, similar to global messages, the documentation is also updated to reflect these changes, and adding/removing fields should be trivial since we can edit it directly through firestore. https://github.com/AtB-AS/docs-private/pull/47/

Reply:
In that case, LGTM 😄